### ISSUE TITLE
setup.py

### ISSUE THREAD

**Issue by aikidoka** [aikidoka/coopdoor#2]

**Comment by javogiraldez** [comment]
Hi, I really like this project for my coop... Any news?
Thanks!

**Comment by aikidoka** [comment]
Yeah, sorry... sick chickens followed by life = _huge delay in progress_. Still planning on getting this put together, but might be a little further delayed. 

The original source project does function with a little work. After you run the ansible config locally, you just have to adjust the sunset data source - or manually change the timer run times (and disable the updater.)
